### ISSUE TITLE
[Question] Do you override initial learning if you fine-tune with a different dataset

### ISSUE THREAD

**Issue by DeFilippis** [minimaxir/gpt-2-simple#187]
I fine-tuned GPT-2 over 12,000 steps on a set of tweets (24k lines).  I noticed that the results, while good, didn't have the depth of content that I was hoping for.

So I re-trained GPT-2 on a separate dataset, using the parameter `restore_from='latest'`, but selecting a different dataset for the `dataset` parameter.  I didn't want to override the personality generated with the original training, so I only trained on the new dataset for 5,000 steps.  

I notice now, however, that any semblance of the original content and writing styles has been overriden in light of the new training data.  I'm wondering if this is to be expected?  Does re-fine-tuning with a different dataset override the initial training weights?   

Should I have just combined the two datasets together and trained on the entirety of that dataset?

**Comment by minimaxir** [comment]
Nope; in fact you can blend datasets together by doing such, albeit it's not an exact science. Janelle Shane had a couple neat outputs with that.
